composer.json

```diff
@@ -12,13 +12,13 @@
     "require": {
         "php": ">=5.6 || ^7.0",
         "sokil/php-mongo": "1.*",
-        "symfony/console": ">=2.0 <5.0",
-        "symfony/yaml": ">=2.0 <5.0"
+        "symfony/console": ">=2.0 <6.0",
+        "symfony/yaml": ">=2.0 <6.0",
+        "squizlabs/php_codesniffer": "^3"
```

Comment:
why this inside `require`? Its not required for production use

Reply:
Soz, my bad, I've missed the `--dev` flag, it's corrected in next commit